### PR TITLE
Add new doc for dummy network

### LINKED ISSUE #2655: Improve documentation for sandbox network.

### PR DESCRIPTION
**Pull request by AurelienFT**
Resolve #2655

### CODE REVIEW

File: docs/technical-doc/dummy-network-generation.rst [hunk 1]
```diff
@@ -1,43 +1,85 @@
...
-.. code-block:: javascript
+  .. code-block:: bash
+    
+    cd massa-node && cargo run --features sandbox
 
-    [
-    "PRI"
-    ]
+The network with your node all start in 10 seconds and you can now interact it with the CLI client like a testnet node.
+If you want to run multiple nodes on your local network you need to use the simulator that use docker all the documentation is here : https://github.com/massalabs/massa-network-simulator
```
Comment (Eitu33) [inline docs/technical-doc/dummy-network-generation.rst:81]:
```suggestion
Your sandbox network containing your node will start after 10 seconds. You can now interact with it through the CLI client in the same way you would with a testnet node.
If you want to run multiple nodes on your local network you should use our advanced simulator. All the documentation is here : https://github.com/massalabs/massa-network-simulator
```

File: docs/technical-doc/dummy-network-generation.rst [hunk 2]
```diff
@@ -1,43 +1,85 @@
...
+    cd massa-client && cargo run
+    wallet_generate_private_key
+
+For the rest of the tutorial we will use theses abreviations:
+
+- `PRIVK` : The private key you just generated
+- `PUBK` : The public key corresponding to PRIVK
+- `ADDR` : The address corresponding to PUBK
+
+Setup your node to use the private key you just generate as its private key and staking key:
```
Comment (Eitu33) [inline docs/technical-doc/dummy-network-generation.rst:35]:
```suggestion
Setup your node to use the private key you just generated as its private key and staking key:
```

File: docs/technical-doc/dummy-network-generation.rst [hunk 3]
```diff
@@ -1,43 +1,85 @@
...
-.. code-block:: javascript
+Clone massa:
 
-    {
-        "ADR": 100
+.. code-block:: bash
+
+    git clone git@github.com:massalabs/massa.git
+
+Compile it with sandbox feature enabled:
```
Comment (Eitu33) [inline docs/technical-doc/dummy-network-generation.rst:16]:
```suggestion
Compile it with the sandbox feature enabled:
```

File: docs/technical-doc/dummy-network-generation.rst [hunk 4]
```diff
@@ -1,43 +1,85 @@
...
+    
+    cd massa-node && cargo run --features sandbox
 
-    [
-    "PRI"
-    ]
+The network with your node all start in 10 seconds and you can now interact it with the CLI client like a testnet node.
+If you want to run multiple nodes on your local network you need to use the simulator that use docker all the documentation is here : https://github.com/massalabs/massa-network-simulator
+
+**On Docker**
```
Comment (Eitu33) [inline docs/technical-doc/dummy-network-generation.rst:83]: Why talk twice about this? You must be really proud of it 😄

Comment (AurelienFT) [reply docs/technical-doc/dummy-network-generation.rst:85]: Done.

**Review by Eitu33 — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**